nubison_model/Service.py

```diff
@@ -19,48 +19,39 @@
 from nubison_model.utils import temporary_cwd
 
 
-def load_nubison_model(
-    mlflow_tracking_uri,
-    mlflow_model_uri,
-    prepare_artifacts: bool = False,
```

Comment:
I don't quite understand the advantages of having the artifacts work in the Service section instead of where they were previously operating